packages/select/src/ux-select.ts

```diff
@@ -163,6 +163,10 @@ export class UxSelect implements UxInputComponent {
     );
   }
 
+  public detached() {
+    this.element.appendChild(this.optionWrapperEl);
```

Comment:
let's extract this method and the one that move the `optionWrapperEl` into 2 methods: 1 to move to body and 1 to move back inside this component.

Reply:
done. I've also unsubscribed on `detached`